modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectEntryLocalServiceImpl.java

```diff
@@ -2205,6 +2222,22 @@ private void _validateAccountEntry(
 				" does not belong to it"));
 	}
 
+	private void _validateExternalReferenceCode(
+			String externalReferenceCode, long companyId,
```

Comment:
Hi Paulo, could you sort the parameters please?

Reply:
okay